docker/Dockerfile.test-ref

```diff
@@ -10,6 +10,18 @@ WORKDIR /app
 RUN samtools faidx ref/Homo_sapiens_assembly38.fasta 'chr1:1-1000000' 'chr2:1-1000000' 'chr10:1-1000000' 'chrX:1-3000000' 'chrY:1-3000000' | sed 's/:1-[0-9]*//' >ref/Homo_sapiens_assembly38.trim.fasta
 RUN samtools faidx ref/Homo_sapiens_assembly38.trim.fasta
 
+RUN curl "https://sano-public.s3.eu-west-2.amazonaws.com/GSA-24v3-0_A2.csv.gz" \
+  -o ref/GSA-24v3-0_A2.csv.gz
+RUN gunzip -c ref/GSA-24v3-0_A2.csv.gz | head -n 8 > ref/GSA-24v3-0_A2.trim.csv \
+  && gunzip -c ref/GSA-24v3-0_A2.csv.gz \
+  | grep -E '(,38,1,)|(,38,2,)|(,38,10,)' \
+  | awk -F',' '$11<1000000' \
+  >> ref/GSA-24v3-0_A2.trim.csv \
```

Comment:
I'm a big fan of these bash pipes but they're not super readable. I comment saying something like "subsample markers in the first gb of chromosomes 1, 2 and 10 plus the first 3 gb of chrs X and Y" would make this more maintainable

Reply:
Also I'm curious why we're only using 1 gb here but 3 for the other test files?

Reply:
Good point about the comment.

I wrote this first, then `tests/data/download.sh`, then edited `tests/data/download.sh` till it worked and forgot to come back here to update to match. 

From setting up the testing so far, I think the first 3 million bases of these five chromosomes is good enough trade off between testing and scale, but we can always change it later if we want.